### PR TITLE
Add Hackathon Dresden

### PR DESCRIPTION
**Pull request by FriederikeZelke**
Add the hackathon in Dresden

### CODE REVIEW

**Comment by fkr** [comment]
I reverted my changes, since it was pointed out to me that @FriederikeZelke wanted to fix this

File: Add Hackathon to Calendar.yml [hunk 1]
```diff
@@ -205,6 +205,7 @@ events:
     begin: 2023-11-08 10:00:00
     duration:
       minutes: 480
-    description: We meet as community to develop, think, collect, and hack together. Details follow soon.
+    description: |
+      We meet as community to develop, think, collect, and hack together. Details follow soon.
       Coordinator: Friederike Zelke <zelke[at]osb-alliance.com>
-    location: "Cloud%Heat, Halle 15, Königsbrücker Straße 96, 01099 Dresden, Germany"
+    location: "CLOUD%HEAT, Halle 15, Königsbrücker Straße 96, 01099 Dresden, Germany"
```
[the remark this answers is not in the document]
Comment (jschoone) [reply Add Hackathon to Calendar.yml]: @FriederikeZelke ping

**Comment by FriederikeZelke** [comment]
Delete team meetings IaaS and IAM for 8th of November.

**Review by jschoone — APPROVED** [review]
/lgtm

**Pull request merged**